x-pack/plugins/canvas/canvas_plugin_src/functions/common/any.ts

```diff
@@ -4,7 +4,13 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export function any() {
+import { FunctionSpec } from '../types';
+
+interface Arguments {
```

Comment:
Maybe `Arguments` in `common/all.ts` was exported to eventually reuse it here? Looks identical